.github/workflows/integration_test.yaml

```diff
@@ -41,12 +40,15 @@ jobs:
       matrix:
         arch:
         - {id: amd64, builder-label: ubuntu-22.04, tester-arch: x64}
+        - {id: arm64, builder-label: ARM64,        tester-arch: ARM64}
         suite: ["k8s", "etcd", "ceph"]
     with:
       identifier: ${{ matrix.arch.id }}-${{ matrix.suite }}
       builder-runner-label: ${{ matrix.arch.builder-label }}
       charmcraft-channel: ${{ needs.charmcraft-channel.outputs.channel }}
-      extra-arguments: ${{needs.extra-args.outputs.args}} -k test_${{ matrix.suite }}
+      extra-arguments: >-
+        ${{needs.extra-args.outputs.args}} -k test_${{ matrix.suite }}
+        ${{ matrix.arch.id == 'arm64' && ' --lxd-containers' || '' }}
```

Comment:
I did some study on a failure on ARM now that we're testing with lxd VMs.

lxd is required to use `/dev/kvm`, but on ARM64 -- `/dev/kvm` may not exist.  I even logged into the machine and tried to add `sudo modprobe kvm` but it fails to load the kernel mode.  

without `/dev/kvm` -- lxd doesn't start the machines and the tests fail

Therefore -- this change allows for lxd-containers on ARM -- but i expect we'd need arm64 to support `/dev/kvm` or don't use lxd as a cloud for arm64